cmd/codeqlExecuteScan_test.go

```diff
@@ -36,7 +36,7 @@ func newCodeqlExecuteScanTestsUtils() codeqlExecuteScanMockUtils {
 }
 
 func TestGetMavenSettings(t *testing.T) {
-	t.Parallel()
+	//t.Parallel()
```

Comment:
we would ideally need a approval from the code ql team , could we create a seperate PR for this first if it fixes the unit test ?

Reply:
reverted it. Will create separate PR later. Hope that checks in this PR will pass :)